cgroup2/manager_test.go

```diff
@@ -147,35 +139,31 @@ func TestSystemdFullPath(t *testing.T) {
 func TestKill(t *testing.T) {
 	checkCgroupMode(t)
 	manager, err := NewManager(defaultCgroup2Path, "/test1", ToResources(&specs.LinuxResources{}))
-	if err != nil {
-		t.Fatal(err)
-	}
-	var procs []*exec.Cmd
-	for i := 0; i < 5; i++ {
+	require.NoError(t, err)
+
+	var (
+		procs    []*exec.Cmd
+		numProcs = 5
+	)
+	for i := 0; i < numProcs; i++ {
 		cmd := exec.Command("sleep", "infinity")
-		if err := cmd.Start(); err != nil {
-			t.Fatal(err)
-		}
-		if cmd.Process == nil {
-			t.Fatal("Process is nil")
-		}
-		if err := manager.AddProc(uint64(cmd.Process.Pid)); err != nil {
-			t.Fatal(err)
-		}
+		err = cmd.Start()
+		require.NoError(t, err)
+		require.NotNil(t, cmd.Process, "process is nil")
+
+		err = manager.AddProc(uint64(cmd.Process.Pid))
+		require.NoError(t, err)
+
 		procs = append(procs, cmd)
 	}
 	// Verify we have 5 pids before beginning Kill below.
 	pids, err := manager.Procs(true)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if len(pids) != 5 {
-		t.Fatalf("expected 5 pids, got %d", len(pids))
-	}
+	require.NoError(t, err)
+	require.Equal(t, numProcs, len(pids), "pid count unexpected")
```

Comment:
testify does have a `Len` method too

```suggestion
	require.Len(t, pids, numProcs, "pid count unexpected")
```

Reply:
ack, swapping to this!